tests/test_vasp_poscar_to_system.py

```
@@ -14,6 +14,30 @@ def setUp(self):
         self.system = dpdata.System()
         self.system.from_vasp_poscar(os.path.join("poscars", "POSCAR.oh.c"))
 
+    def test_move_flags(self):
+        expected = np.array([[[True, True, False], [False, False, False]]])
+        self.assertTrue(np.array_equal(self.system["move"], expected))
+
+
+class TestPOSCARCart(unittest.TestCase):
```

Ruff

Redefinition of unused `TestPOSCARCart` from line 12

(F811)